migrate/migrate.go

```diff
@@ -0,0 +1,13 @@
+package migrate
+
+type Migrator interface {
```

Comment:
Unless mistaken, it seems this interface isn't used anywhere, in which case, we should remove it.